src/features/classification/__tests__/Classification.test.js

```diff
@@ -57,6 +60,18 @@ describe('features/classification/Classification', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should render classification last modified information when provided and message style is inline', () => {
+        const wrapper = getWrapper({
+            name: 'Confidential',
+            definition: 'fubar',
+            intl,
+            messageStyle: 'inline',
+            modifiedAt: '2020-07-16T00:51:10.000Z',
+            modifiedBy: 'A User',
+        });
+        expect(wrapper).toMatchSnapshot();
```

Comment:
We should use explicit assertions instead of snapshots when possible.

Reply:
Sure, I already have a classname on the `<p>` might as well use it to verify existence.

Reply:
Snapshot still needed? Are there other assertions we should add to replace it?